<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linear Interpolation Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ME7.jpg</normaloff>ME7.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>연세제목체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Linear Interpolation Calculator</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_but">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>연세소제목체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
    <property name="autoRepeatInterval">
     <number>98</number>
    </property>
   </widget>
   <widget class="QPushButton" name="End_but">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>연세소제목체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
    <property name="autoRepeatInterval">
     <number>98</number>
    </property>
   </widget>
   <widget class="QLabel" name="Title_label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>연세제목체</family>
     </font>
    </property>
    <property name="text">
     <string>by_2019145120 서기훈</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Eagle_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>파란 독수리.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Title_label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>연세제목체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(선형 보간 계산기)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>Title_label</zorder>
   <zorder>Start_but</zorder>
   <zorder>End_but</zorder>
   <zorder>Title_label_2</zorder>
   <zorder>Title_label_3</zorder>
   <zorder>Eagle_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionasdf_2">
   <property name="text">
    <string>asdf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
